src/middlewared/middlewared/validators.py

```diff
@@ -24,11 +25,21 @@ def __call__(self, value):
 
 
 class IpAddress:
```

Comment:
IpAddress is a IpAddress, IP:port is something else.

I think it makes sense to logically separate those.

Reply:
I am not even sure if it makes sense to have this reusable since there isnt anywhere else we use this and there could be different constraints for each. Perhaps just do the validation on netdata?